### DISCUSSION TITLE
Help with Vivo SP

### DISCUSSION THREAD

**Discussion by Edu-ST** [fabianishere/udm-iptv#152] [Q&A] [answered]
I need some help or direction to follow, please...

The setup was fine (UDM Pro), vlan for iptv created, I connect to tv provider (VIVO SP) but the TV broadcast hangs after 15 seconds, so I just need to change the channel in the Menu and it plays again for another 15 seconds and freezes. As it was waiting for a data request to send one more packet, and then stop until another request.
What's happening? Is there something wrong with the vlan configuration?
Thank you !!

**Comment by cawv** [comment]
Got any luck with Vivo Fiber? 
I'm on the same boat, trying to use my new UDM Pro to route both IPTV and internet in the same cable.
Share your experience.

**Comment by Edu-ST** [comment]
Yeap, but internet only..
I changed the original modem for UFiber connected as a bridge, authenticated by UDM Pro and VLAN 10 created. So, the internet is alive..
Around IPTV, it's connected, but it's unconnected a few seconds after.
When I change the Decoder channel, it works again, but for no more than a few seconds.
I'm standing here...

**Reply by apedroheringer** [reply]
Hi, my setup is Fiber -> TP-Link tx6610 -> udm pro. I don't have any problema running this script.
I've tried using a ufiber loco (profile 3) instead of the tplink but the tv freezes every 10 seconds.

**Reply by Edu-ST** [reply]
> Hi, my setup is Fiber -> TP-Link tx6610 -> udm pro. I don't have any problema running this script. I've tried using a ufiber loco (profile 3) instead of the tplink but the tv freezes every 10 seconds.

@apedroheringer  this way, is IPTV running as well?   Yeap, I use UFiber and no way.  I will order a tx6610 unit.  Could you share the setup to 6610?  Thank you!

**Reply by apedroheringer** [reply]
Yes, you only have to setup the dns nameservers the vlan. Without doing that Vivo play / on demand won't work.
I can share but not now, tomorow night or on wednesday afternoon Ill have time for that

**Reply by Edu-ST** [reply]
> Yes, you only have to setup the dns nameservers the vlan. Without doing that Vivo play / on demand won't work. I can share but not now, tomorow night or on wednesday afternoon Ill have time for that

Hi @apedroheringer as soon as you can, please share your TX6610 setup please...  and if you can to share your VLAN config also because I think that my VLAN could have some gap.
I receive my new TX by today. Tks !!!

**Reply by apedroheringer** [reply]
Vc mora em sp (telefonica) ou outro estado (gvt)? pelo que sei as vlans da area da gvt são diferentes. O script usa as vlans e as rotas da vivo sp (que era telefonica). Vou ver agora a configuração do tplink e te falo

**Reply by Edu-ST** [reply]
> Vc mora em sp (telefonica) ou outro estado (gvt)? pelo que sei as vlans da area da gvt são diferentes. O script usa as vlans e as rotas da vivo sp (que era telefonica). Vou ver agora a configuração do tplink e te falo

SP capital..  e criei uma VLAN 20, mas não sei se faltou algo na configuração.  Hoje, usando o UFiber eu ja consigo conectar, apesar da falha que vc tb sofreu de desconectar a cada 10s. Mas não sei se isso já garante que a VLAN que criei estaria 100% correta. Obrigado

**Reply by apedroheringer** [reply]
Vamos lá, voce acessa o modem vivo em http://192.168.15.1/device-management-statistics.asp, o login é admin e a senha está colada embaixo do modem. Nessa pagina voce vai pegar o Número de Série GPON: e o SLID:

Na configuração do tplink vc vai colocar o slid no campo password (esse numero é por região, é fixo, ja trocaram meu modem e ele nao mudou).
O numero de série voce coloca no gpon sn. O porem é que tem que ser em hexadeximal, esse numero de serie da vivo os 4 primeiros numeros sao ascii, o resto hexadeximal. Por exemplo, supondo que o meu seja TLCM00B427E6 preciso converter esse TLCM para hexa. Vai nesse site https://www.rapidtables.com/convert/number/ascii-to-hex.html e converta, TLCM virou 544C434D, entao vou colocar como numero de serie 544C434D00B427E6. Quando trocaram meu modem eu nao precisei mudar esse numero de serie, o numero de serie que tá no meu meu tplink nao bate com meu modem da vivo atual, funciona normalmente. Mas enfim, quando configurei lá atras estava com o correto.
na vlan setting vc coloca como transparent, pra ele passar a vlan 10, 20 e 30 pro udm. Vou pegar as configuracoes do udm e ja te passo, mas pelo que vc disse devem estar corretas....

**Reply by apedroheringer** [reply]
Na vlan que criei para o iptv ativei o IGMP Snooping e Multicast DNS, e coloquei o dhcp dns server 177.16.30.67 e 177.16.30.7
Com a vlan separada voce nao vai conseguir usar o youtube como chromecast por exemplo. Eu cheguei uma epoca a deixar tudo na mesma vlan colocando uma regra de no boot ele dar esses dns servers pros STBs mas acabei tirando, fiquei com medo de floodar a rede, wifi...

**Reply by apedroheringer** [reply]
> > Vc mora em sp (telefonica) ou outro estado (gvt)? pelo que sei as vlans da area da gvt são diferentes. O script usa as vlans e as rotas da vivo sp (que era telefonica). Vou ver agora a configuração do tplink e te falo
> 
> SP capital.. e criei uma VLAN 20, mas não sei se faltou algo na configuração. Hoje, usando o UFiber eu ja consigo conectar, apesar da falha que vc tb sofreu de desconectar a cada 10s. Mas não sei se isso já garante que a VLAN que criei estaria 100% correta. Obrigado

eu nao faço ideia do pq a tv parar de funcionar no ufiber... queria tanto ele, nao gosto do tplink ser vertical....

**Reply by apedroheringer** [reply]
Enfim, na configuração de internet voce configura normal. vlan tag 10, login cliente@cliente, senha cliente
O script do fabianishere que vai fazer a outra conexao na vlan 20 da vivo, puxando a conexao do iptv, e jogar para a rede local vlan que voce especificar na configuração do script. Dai taggear as portas que os stbs estão conectados para usarem a vlan que está recebendo a conexão da vlan20/iptv

**Reply by Edu-ST** [reply]
> Vamos lá, voce acessa o modem vivo em http://192.168.15.1/device-management-statistics.asp, o login é admin e a senha está colada embaixo do modem. Nessa pagina voce vai pegar o Número de Série GPON: e o SLID:
> 
> Na configuração do tplink vc vai colocar o slid no campo password (esse numero é por região, é fixo, ja trocaram meu modem e ele nao mudou). O numero de série voce coloca no gpon sn. O porem é que tem que ser em hexadeximal, esse numero de serie da vivo os 4 primeiros numeros sao ascii, o resto hexadeximal. Por exemplo, supondo que o meu seja TLCM00B427E6 preciso converter esse TLCM para hexa. Vai nesse site https://www.rapidtables.com/convert/number/ascii-to-hex.html e converta, TLCM virou 544C434D, entao vou colocar como numero de serie 544C434D00B427E6. Quando trocaram meu modem eu nao precisei mudar esse numero de serie, o numero de serie que tá no meu meu tplink nao bate com meu modem da vivo atual, funciona normalmente. Mas enfim, quando configurei lá atras estava com o correto. na vlan setting vc coloca como transparent, pra ele passar a vlan 10, 20 e 30 pro udm. Vou pegar as configuracoes do udm e ja te passo, mas pelo que vc disse devem estar corretas....

Assim que acessar meu UFiber vou confirmar mas se não engano os campos que usei nele são:
SLID
GPON pass
PLOAM
e coloquei o SLID e passw o mesmo como deve ser, e no PLOAM me indicaram pra repetir, ou seja, não usei o SN em lugar nenhum.. talvez até valha testar..

**Reply by Edu-ST** [reply]
> Na vlan que criei para o iptv ativei o IGMP Snooping e Multicast DNS, e coloquei o dhcp dns server 177.16.30.67 e 177.16.30.7 Com a vlan separada voce nao vai conseguir usar o youtube como chromecast por exemplo. Eu cheguei uma epoca a deixar tudo na mesma vlan colocando uma regra de no boot ele dar esses dns servers pros STBs mas acabei tirando, fiquei com medo de floodar a rede, wifi...

Vc chegou a criar no DHCP dessa VLAN a opção: 
Name | Code | Type | Value
IPTV-Class | 60 | Text | IPTV_RG

**Reply by Edu-ST** [reply]
> eu nao faço ideia do pq a tv parar de funcionar no ufiber... queria tanto ele, nao gosto do tplink ser vertical....

Pois é...   eu tb queria muito... a qualidade de construção dele é incrível.
Se funcionar com o TX6610 como vc conseguiu fazer, se quiser te empresto o UFiber para tentar alguma coisa nova

**Reply by Edu-ST** [reply]
> Enfim, na configuração de internet voce configura normal. vlan tag 10, login cliente@cliente, senha cliente O script do fabianishere que vai fazer a outra conexao na vlan 20 da vivo, puxando a conexao do iptv, e jogar para a rede local vlan que voce especificar na configuração do script. Dai taggear as portas que os stbs estão conectados para usarem a vlan que está recebendo a conexão da vlan20/iptv

Isso que ia mesmo perguntar... sobre taggear a porta do STB..   tirando isso a vlan10 para mim está 100% já com o UFiber
Como eu já rodei o script dele, será que é melhor primeiro executar o Remove primeiro, para dai refazer...   ou posso rodar de novo direto?

**Reply by apedroheringer** [reply]
pode rodar o script dnovo direto... o ufiber tentei com aquele https://github.com/Unifi-Tools/UFiber.Configurator e nao mudou muita coisa na iptv

**Comment by cawv** [comment]
Figured out only today.
Main problem is around UDM lack of IGMP Proxy.
USG had it but no escuse why still not implemented in UDM.
Saw on the internet one solution using exactly the same Askey router from Movistar Spain, they have used managed switches from Unifi.
At the end both were working simultaneously in the same cable only splitting the Vlans at the end with small flex switches in the rooms.
Nice elegant solution.


Christiano Vilhena

 Enviado via iPhone

**Reply by fabianishere** [reply]
This project should help you setup IGMP Proxy on the UDM.

**Comment by Edu-ST** [comment]
@fabianishere can I capture any log to get look?
which one should be better?
i created the specific vlan to IPTV and ran your script

![image](https://user-images.githubusercontent.com/63136506/187046532-f36366a4-8cad-4fa1-86fb-74d0dde419ff.png)

**Reply by fabianishere** [reply]
Could you share the output of `udm-iptv-diag` as well?

**Comment by cawv** [comment]
Do the IGMP proxy works with Vivo Mediaroom setup? Changed mine to Open platform trying to get a better soluttion but no clue......

**Reply by fabianishere** [reply]
Perhaps @apedroheringer knows more about this?

**Reply by apedroheringer** [reply]
Here my logs

root@ubnt:/# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 20 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 172.28.0.0/14  201.0.52.0/23  200.161.71.0/24  177.16.0.0/16
LAN Interfaces: br20
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
43: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.154.188.196/20 scope global iptv
       valid_lft forever preferred_lft forever
10.154.176.0/20 proto kernel scope link src 10.154.188.196
172.28.0.0/14 via 10.154.176.1 metric 243
177.16.30.0/23 via 10.154.176.1 metric 248
200.161.71.40/30 via 10.154.176.1 metric 245
200.161.71.46/31 via 10.154.176.1 metric 246
200.161.71.48/31 via 10.154.176.1 metric 247
201.0.52.0/23 via 10.154.176.1 metric 244
=== Service Logs ===
Aug 29 13:42:24 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machine.
Aug 29 13:42:24 ubnt udm-iptv[63]: Obtaining IP address for VLAN interface
Aug 29 13:42:24 ubnt udm-iptv[63]: udhcpc (v1.22.1) started
Aug 29 13:42:24 ubnt udm-iptv[63]: Sending discover...
Aug 29 13:42:24 ubnt udm-iptv[63]: Sending select for 10.154.188.196...
Aug 29 13:42:24 ubnt udm-iptv[63]: Lease of 10.154.188.196 obtained, lease time 7200
Aug 29 13:42:24 ubnt udm-iptv[63]: NATing IPTV network ranges (if necessary)
Aug 29 13:42:24 ubnt udm-iptv[63]: Setting up igmpproxy
Aug 29 13:42:25 ubnt udm-iptv[63]: Starting igmpproxy
Aug 29 21:30:44 ubnt igmpproxy[63]: MRT_DEL_MFC; Errno(2): No such file or directory

The tv works without any error

**Reply by apedroheringer** [reply]
I think that my plataform is open, my STBs are Arris

**Comment by apedroheringer** [comment]
Ah sim, criei isso sim...

Em sex., 2 de set. de 2022 16:35, Eduardo ***@***.***>
escreveu:

**Comment by apedroheringer** [comment]
Então, o meu ufiber loco configurei naquele profile 3. Não lembro se fiz
mais testes na época, acho que vou brincar disso agora rsrs

Em sex., 2 de set. de 2022 16:31, Eduardo ***@***.***>
escreveu:

**Comment by apedroheringer** [comment]
Depois que ele roda o script ele é iniciado sempre com o udm.... Só precisa
rodar o script pra jogar o IPTV pra sua vlan interna da IPTV... Mas se vc
vc tinha configurado isso antes acho que não precisa

Em sex., 2 de set. de 2022 16:42, Eduardo ***@***.***>
escreveu:

**Comment by Edu-ST** [comment]
@apedroheringer  se não tiver mais o UFiber, te empresto o meu assim que ligar o tx6610 se quiser tentar algo diferente..  eu ainda vou continuar tentando até achar um jeito de usar ele rsrsrs..   só vou colocar o tx agora, para aproveitar a tv enqto isso..   meu uso principal é para streaming de audio hi end, e vou te falar, com o ufiber melhorou muito !!!!

**Comment by apedroheringer** [comment]
Eu ainda tenho o meu... vi isso aqui agora
https://github.com/Unifi-Tools/UFiber.Configurator acho ue vou dar uma lida
e ver se consigo algo diferente

Em sex., 2 de set. de 2022 às 16:59, Eduardo ***@***.***>
escreveu:

**Reply by apedroheringer** [reply]
configurei o ufiber a partir dai mas continuou travando a tv depois de uns 10 segundos :(

**Reply by Edu-ST** [reply]
putz...  ainda vou tentar e ver se falo com uns amigos que trabalharam lá, se contatam alguém para pedir ajuda..  não vou desistir rsrs
seu tx6610 está com essa versão de firmware: 3.1.4 build 181115 Rel.51843n
fui pegar no site para atualizar, mas tem uma versão de 2015 rsrs talvez a que veio seja mais atual até..  estranho ne

**Reply by apedroheringer** [reply]
meu tplink é hardware v4.0, firmware 3.1.4 build 180122 rel.50640n, fui ver no site e realmente é uma versao mais antiga, bem estranho isso rsrs. nao vou procurar mexer no firmware pra atualizar pq tá funcionando redondo aqui...

**Reply by Edu-ST** [reply]
> meu tplink é hardware v4.0, firmware 3.1.4 build 180122 rel.50640n, fui ver no site e realmente é uma versao mais antiga, bem estranho isso rsrs. nao vou procurar mexer no firmware pra atualizar pq tá funcionando redondo aqui...

Louco isso né rsrs
Eu ainda não mexi tb, vou experimentar assim.. mas vou tentar pegar a versão do teu por garantia

**Comment by Edu-ST** [comment]
ao taggear a porta a ser usada pelo STB, vc criou um Perfil Custom no Switch Ports?  
ou apenas no Port Management, ao abrir o Port Profile da porta do STB selecionou em Network da VLAN criada?


![image](https://user-images.githubusercontent.com/63136506/188251108-195353fc-90f7-4e8e-9402-6983d0bfd0a1.png)




Até criei um novo perfil com:
Native Network = Default (minha network default) ou aqui já indico a VLAN20?
Tagged Network = vlan20

depois de criado esse perfil, é o que uso na Porta que vou conectar o STB, certo?

![image](https://user-images.githubusercontent.com/63136506/188250188-ce36ff13-4dfb-4d07-a9c1-b2e099891b6e.png)

**Reply by apedroheringer** [reply]
eu nao criei perfil nenhum, coloquei da lista networks a vlan que criei, IPTV, direto

**Reply by apedroheringer** [reply]
![image](https://user-images.githubusercontent.com/67462182/188282217-a26dd3e0-1c76-46b3-8211-710b630ec560.png)

**Comment by jotala** [comment]
I’ve been trying to set up IPTV for more than a year.
I read every discussion and issue, and I thought it was a DHCP option 43 and 121 problems. #96 
It is my first time using GitHub, excuse me for any mistakes. 

I think I found out:

Maybe  @apedroheringer setup uses an old IPTV system from Vivo ISP, called Media Room, that’s why his setup is working.
Like @cawv my system is Open Platform.

On the ISP router, it shows on the WAN interfaces, and it VLANs

600 Internet
601 VoIP
602 IPTV
4000 Multicast

(Mine VLAN is different from the others, because Vivo differentiated VLAN based on regions, this is not an issue as successfully got the routes and IP on VLAN 602)

What I saw on Wireshark:
IPTV goes thru the IP gathered from DHCP on 602
Multicast origins are from 10.0.0.1 on VLAN 4000

I don’t know if is the same problem from “PostTV” #86 where the guy is using “eth8.35” who needed the IPTV originates from 10.10.10.10



With the current configuration, I was able to get the IP and routes, the Live TV works for about 10 seconds then freezes, if I change the channel the behavior repeats. 
I can watch and use the DVR function to rewind a show and it never freezes as long I’m behind the live streaming. (I think this is due to unicast working properly)



@fabianishere is it possible to have two Vlans on the IPTV interface, or segregate the multicast VLAN?


If you want I could share the file from Wireshark, also the routes, etc.


```
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 602 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 0.0.0.0/0
LAN Interfaces: br6
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
73: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.182.A.BBB/20 scope global iptv
       valid_lft forever preferred_lft forever
10.182.0.0/20 proto kernel scope link src 10.182.A.BBB 
172.28.0.0/16 via 10.182.0.1 metric 273 
177.16.30.0/23 via 10.182.0.1 metric 275 
200.161.71.40/30 via 10.182.0.1 metric 276 
200.161.71.46/31 via 10.182.0.1 metric 277 
200.161.71.48/31 via 10.182.0.1 metric 278 
201.0.52.0/23 via 10.182.0.1 metric 274 
=== Service Logs ===
Sep 06 15:24:03 ubnt udm-iptv[9091]: Inserted route table entry for 239.130.2.0 on VIF #0
Sep 06 15:24:03 ubnt udm-iptv[9091]: Joining group 239.130.2.0 on interface iptv
Sep 06 15:24:04 ubnt udm-iptv[9091]: RECV V2 member report   from 192.168.15.244  to 239.131.0.0
Sep 06 15:24:04 ubnt udm-iptv[9091]: Updated route entry for 239.131.0.0 on VIF #0
Sep 06 15:24:04 ubnt udm-iptv[9091]: RECV Leave message      from 192.168.15.244  to 224.0.0.2
Sep 06 15:24:04 ubnt udm-iptv[9091]: Leaving group 239.131.0.0 on interface iptv
Sep 06 15:24:04 ubnt udm-iptv[9091]: RECV Membership query   from 192.168.15.1    to 239.131.0.0
Sep 06 15:24:04 ubnt udm-iptv[9091]: RECV V2 member report   from 192.168.15.244  to 239.131.0.0
Sep 06 15:24:04 ubnt udm-iptv[9091]: Updated route entry for 239.131.0.0 on VIF #0
Sep 06 15:24:04 ubnt udm-iptv[9091]: Joining group 239.131.0.0 on interface iptv
```

**Reply by fabianishere** [reply]
No, IGMP Proxy only works with a single upstream, so having multiple upstream VLANs is not possible. Are you sure you need two upstream VLANs?

**Reply by jotala** [reply]
I don't. This is my first time using Wireshark.
Can i send you the file privately, since I don't know if it contains any sensitive information?

I'll do anything. I'm working on this for 2 weeks every single day. and for a year some days a month.

**Reply by apedroheringer** [reply]
Você usa qual gpon? Se eu uso o UFiber-loco minha iptv trava em 10 segundos, se eu uso o TP-link TX-6610 funciona normalmente do modo que colocamos no script.
Fora de SP realmente a Vivo usa outras Vlans, por isso no script coloquei como Vivo SP (Vlan 10 internet, Vlan 20 tv e Vlan 30 telefone, que o script nao cobre)
Meus STB são Arris, o gravador não é multiroom, isso é caracteristica da OpenTV não? Se não você sabe me dizer onde vejo se uso mediaroom ou opentv?

**Reply by jotala** [reply]
Sorry for writing in Portuguese Fabianishere, the talk bellow is only clarification about config and hardware. Nothin you can help. ;)

> Você usa qual gpon? Se eu uso o UFiber-loco minha iptv trava em 10 segundos, se eu uso o TP-link TX-6610 funciona normalmente do modo que colocamos no script.

Uso TP-Link ZX-003G3 (modelo mais recente do TX-6610) como conversor de ONT. 
Nas configurações, coloquei modo transparente. Assim feito a conexão PPPOE usa a VLAN Tag 600 no UDM-PRO.

> Meus STB são Arris, o gravador não é multiroom, isso é caracteristica da OpenTV não? Se não você sabe me dizer onde vejo se uso mediaroom ou opentv?

Não sei se isso significa ser Open Plataforma.
Meu STB é Arris VIP4242H.
Acho que o media room usava o decoder arredondado como o IPH8000

Vou fazer um apanhando com todas fotos de rota, configurações do decoder original. Para ajudar a comparar.

@apedroheringer já que o seu é o único funcionando, deixa suas configurações do Firewall também.
Quando fizer o Apanhado com tudo, a gente compara o que aparece no seu router da Vivo com o resto.
A propósito a gente tá no grupo do Telegram. Mas melhor falar aqui para registrar. 

Obrigado por tentar ajudar mesmo o seu funcionando.

**Reply by apedroheringer** [reply]
Eu já tive mediaroom há muitos anos, era decoder cisco/pace. Voltei a assinar a iptv esse ano ou ano passado pra testar exatamente esse script e deu certo. Dessa vez colocaram o gravador arris vip5262, o outro humax ptt-100, até onde sei esses modelos são opentv e a vivo parou de instalar mediaroom tem tempo, tem até migrado clientes antigos.
As rotas para por no script eu peguei aqui: https://github.com/offspringer/unifi-usg-vivo-fibra
Cara, pela coisa que voce falou dos 10 segundos eu realmente acho que a ont deve ter algo de diferente... pq tenho um uf-loco que se uso fica exatamente como vc relatou, ja o tplink tx6610 nao.
Vou tirar foto das minhas regras de firewall e mandar agora no telegram, se voce precisar de mais alguma info ou foto de alguma regra es especifico me avisa...

**Comment by Edu-ST** [comment]
O mesmo comportamento de freeze em 10seg que enfrentava usando o UFiber Loco.
Vou testar a troca pelo TX6610 amanhã ou no final de semana e reporto o que deu... agradeço de novo o @apedroheringer 
@jotala  Qual é esse grupo no Teleg que falaram?  Está aberto?

**Reply by apedroheringer** [reply]
https://t.me/+BQiz25jZaJVhOWFh

**Reply by apedroheringer** [reply]
> O mesmo comportamento de freeze em 10seg que enfrentava usando o UFiber Loco. Vou testar a troca pelo TX6610 amanhã ou no final de semana e reporto o que deu... agradeço de novo o @apedroheringer @jotala Qual é esse grupo no Teleg que falaram? Está aberto?

Espero que de certo! Vai tirar nossa duvida

**Comment by jotala** [comment]
> Vou testar a troca pelo TX6610 amanhã ou no final de semana e reporto o que deu

Por favor reporte. Estou quase comprando. Vê se faz diferença do TP-Link ZX-003G3

**Reply by Edu-ST** [reply]
Pode deixar,  se der tempo farei amanhã ou no max nesse fds.
O comportamento hj é, assim que trava eu mudo de canal e funciona por mais alguns segundos, trava, mudo de novo e por aí vai  . ansioso para resolver isso..  mas o que queria mesmo era que funcionasse com o UFiber rsrs

**Reply by apedroheringer** [reply]
o ufiber é exatamente assim aqui

**Comment by Edu-ST** [comment]
Antes eu havia comprado o UFiber Wifi, que tem 4 portas LAN e iria tentar sair IPTV direto dele...  mas como não havia evoluido como agora,  devolvi

**Comment by apedroheringer** [comment, marked as answer]
@fabianishere @Edu-ST was able to make it work using the same gpon I use... I´ll edit the readme indicating people to use this gpon (TP-Link TX6610

**Comment by 0x3333** [comment]
Hi everyone, I'm trying to enable IPTV from a ODI DFP-34X-2C3 in Brazil Vivo SP(Region 1).

I'm able to connect to the internet, but IPTV freezes after 10/15 seconds. Did anyone tested this GPON? Is there any special configuration in the TX6610?

**Reply by ffichman** [reply]
Você conseguiu resolver? Estou com mesmo problema em outra GPON

**Comment by abelchior77** [comment]
Hi everyone good night. IPTV is running fine using ethernet cable but not working through Wifi (I really dont know if anyone are using wi-fi decoder). 

There's my config and IPTV diag. Thank You in advice.

GPON - TPLink XZ000 GT
Gateway Cloud Gateway Max
WiFi U6 Pro
STB: Arris VIP4342W

root@Cloud-Gateway-Max:/# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth4
WAN VLAN: 20 (dev iptv)
WAN DHCP: true (options "43")
WAN Ranges: 172.28.0.0/14  201.0.52.0/23  200.161.71.0/24  177.16.0.0/16
LAN Interfaces: br20
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
19: iptv@eth4: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.123.0.97/20 scope global iptv
       valid_lft forever preferred_lft forever
10.123.0.0/20 proto kernel scope link src 10.123.0.97
172.28.0.0/14 via 10.123.0.1 metric 219
177.16.30.0/23 via 10.123.0.1 metric 222
200.161.71.40/30 via 10.123.0.1 metric 221
200.161.71.46/31 via 10.123.0.1 metric 224
200.161.71.48/31 via 10.123.0.1 metric 223
201.0.52.0/23 via 10.123.0.1 metric 220
=== Service Logs ===
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444422]: Obtaining IP address for VLAN interface
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444436]: udhcpc: started, v1.30.1
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444436]: udhcpc: sending discover
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444436]: udhcpc: sending select for 10.123.0.97
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444436]: udhcpc: lease of 10.123.0.97 obtained, lease time 21600
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444422]: NATing IPTV network ranges (if necessary)
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444422]: Creating static routes (if necessary)
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444422]: Setting up IGMP Proxy
Oct 08 02:03:38 Cloud-Gateway-Max udm-iptvd[444422]: Using improxy...
Oct 08 02:03:39 Cloud-Gateway-Max udm-iptvd[444422]: Starting IGMP Proxy

**Reply by abelchior77** [reply]
@fabianishere / @apedroheringer any idea?

**Reply by abelchior77** [reply]
@Edu-ST vc conseguiu resolver esse problema. O decoder via cabo ethernet vai legal mas no Wifi ta tendo o mesmo sintoma q vc reportou.

**Comment by apedroheringer** [comment]
Hi Alexandre, eu troquei da vivo pra claro há alguns anos.

Em qui., 9 de out. de 2025, 13:59, Alexandre Belchior <
***@***.***> escreveu:

**Reply by abelchior77** [reply]
Obrigado pelo retorno Pedro

**Comment by apedroheringer** [comment]
Eu estou vendo aqui, na minha época os decoders não tinham wi-fi 😅 não sei
como ajudar mesmo...

Em qui., 9 de out. de 2025, 19:07, Alexandre Belchior <
***@***.***> escreveu:

**Reply by abelchior77** [reply]
Pois e, nao achei nada a respeito

**Comment by ffichman** [comment]
Alguém conseguiu nas novas versões??
meu está dando o erro abaixo:

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 20 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 172.28.0.0/14  201.0.52.0/23  200.161.71.0/24  177.16.0.0/16
LAN Interfaces: br20
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Nov 14 15:13:27 NEXTTHOUSE IMPROXY[2284454]: FATAL[init_interface@251]: exiting.....
Nov 14 15:13:27 NEXTTHOUSE IMPROXY[2284454]: ERROR[main@619]: exiting.........
Nov 14 15:13:27 NEXTTHOUSE systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Nov 14 15:13:32 NEXTTHOUSE systemd[1]: udm-iptv.service: Scheduled restart job, restart counter is at 32.
Nov 14 15:13:32 NEXTTHOUSE systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Nov 14 15:13:32 NEXTTHOUSE systemd[1]: Started IPTV support for the UniFi Dream Machine.
Nov 14 15:13:32 NEXTTHOUSE udm-iptvd[2285548]: Device iptv already exists.. Deleting device
Nov 14 15:13:32 NEXTTHOUSE udm-iptvd[2285548]: Obtaining IP address for VLAN interface
Nov 14 15:13:32 NEXTTHOUSE udm-iptvd[2285562]: udhcpc: started, v1.30.1
Nov 14 15:13:32 NEXTTHOUSE udm-iptvd[2285562]: udhcpc: sending discover

**Reply by ffichman** [reply]
Alguém ja teve o problema de funcionar o canal por 5-10 segundos e parar?

**Reply by 0x3333** [reply]
Sim. Pra mim deve ser o gpon sfp. Pq li
Que um da tplink funcionou. Mas não fui atrás.

**Comment by ffichman** [comment]
Meu GPON está configurado da seguinte maneira e mesmo assim os canais funcionam por 10s e param
<img width="831" height="434" alt="Captura de Tela 2025-11-30 às 14 25 44" src="https://github.com/user-attachments/assets/39baf36e-7e72-419c-a353-2fadb16a81ab" />

**Reply by CharmiUwU** [reply]
which firmware do you have?

**Reply by ffichman** [reply]
> which firmware do you have?

![image](https://github.com/user-attachments/assets/8422a058-4946-48bd-b30c-3b217dbbfaba)

**Reply by CharmiUwU** [reply]
Sorry, on the ODI stick

**Reply by ffichman** [reply]
> Sorry, on the ODI stick

[Uploading M110_sfp_ODI_240202_Extract_Tar_File.zip…]()
